src/components/tabs/example-tabs-details.njk

```diff
@@ -134,9 +134,9 @@
                 {
                     "id": "video-embed",
                     "title": 'Embed',
-                    "hiddenSpan": 'code for Census 2021 promotional TV advert video',
+                    "hiddenSpan": 'code for Transforming the way we produce statistics | Explaining the Dynamic Population Model | BSL',
                     "content": '<h3 class="ons-u-d-no@s">Embed</h3>
-                        <pre class="language-html"><code>' ~ '<iframe width="560" height="315" src="https://www.youtube.com/embed/_EGJlvkgbPo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>' | e ~ '</code></pre>'
+                        <pre class="language-html"><code style="white-space: pre-wrap">' ~ '<iframe width="560" height="315" src="https://www.youtube.com/embed/OwYVwPPxPj4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>' | e ~ '</code></pre>'
```

Comment:
I think we should be avoiding inline css and probably best to set it to "normal"